docs/snippets/common/component-story-mdx-story-by-name.mdx.mdx

```diff
@@ -1,4 +1,8 @@
 ```md
+<!-- Button.stories.mdx --->
+
+<!--- This is your Story template function, shown here in React -->
```

Comment:
Can we put this above the `export const Template = ` line?

Reply:
Updated with the latest commit